### PR TITLE
Phone field displays and is resized

### PR DESCRIPTION
**Pull request by AshlandWest**

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+          root: ["./"],
           alias: {
-            Screens: './src/screens',
-            Store: './src/store',
-            Components: './src/components',
-            Lib: './src/lib',
+            Screens: "./src/screens",
+            Store: "./src/store",
+            Components: "./src/components",
+            Lib: "./src/lib"
```
Comment (luxx) [inline babel.config.js:14]: please leave the trailing comma, it makes it easier to add new items, and clutters the diff

File: src/screens/ProfileScreen.js [hunk 2]
```diff
@@ -1,37 +1,29 @@
...
 import { setUserPhone } from "Store/actions";
 
-import { UserPhoneInput } from "Components/user-phone-input"
+import UserPhoneInput from "Components/user-phone-input";
 
 const styles = StyleSheet.create({
   container: {
-    backgroundColor: "#fff",
-  },
+    backgroundColor: "#fff"
```
Comment (luxx) [inline src/screens/ProfileScreen.js:11]: please leave trailing commas for all lists

**Review by luxx — APPROVED** [review]

File: babel.config.js (hunk 1 above)
Comment (AshlandWest) [reply babel.config.js:14]: Sorry, I think that's my code formatter. I'll review the settings.

File: src/screens/ProfileScreen.js (hunk 2 above)
Comment (AshlandWest) [reply src/screens/ProfileScreen.js:11]: Sorry, I think that's my code formatter. I'll review the settings.

**Pull request merged**